docs/legacy-account-migration.md

```diff
@@ -0,0 +1,25 @@
+# Legacy Account Migration
+
+## How to check an accounts current balance and other information
+
+Account balances migrated from Creditcoin 1.7 at block NNNN. Your starting balance on Creditcoin 2.0 should match your Creditcion 1.7 block NNNN balance
+
+In order to check your Creditcoin 2.0 balance you can run the following command:
+
+   ``` shell
+   docker run jacogr/polkadot-js-tools api --ws wss://<rpc-node> query.system.account <SS58 Address>
+   ```
+
+## How to migrate balances that couldn't be automatically migrated
```

Comment:
Question: Does CC 1.7 shut down or blacklist external nodes after CC 2.0 launch?  What happens for miners which have unmigrated balances but  they stay connected to 1.7 ? Do they lose some of the rewards (after the launch date) or do they get all of their rewards whenever they claim them? 

Similar scenario for already migrated balances. What if a miner keeps their nodes connected to both 1.7 and 2.0 ? 

Maybe worth documentation/release note how the migration process will take place.

Reply:
After N block (TBD) miner's will not be credited with any rewards from the 1.7 network, but they will not be forced to shutdown. A snapshot will be taken of the 1.7 network at that point in time and will serve as the genesis account balances on the 2.0 network (except for the ones that need to be "claimed". We will give more details on this process.

Reply:
> We will give more details on this process.

Do you mean externally, e.g. outside of this repo/file or do you mean you will amend the current documentation? 

Either one is fine by me, just want to align expectations.